app/views/articles/_sidebar_nav.html.erb

```diff
@@ -48,7 +48,7 @@
       <% end %>
     </div>
     <div class="sidebar-nav-block-inner">
-      <div id="sidebar-nav-followed-tags" class="sidebar-nav-followed-tags"></div>
+      <div id="sidebar-nav-followed-tags-ctn"></div>
```

Comment:
We are using Preact 8.x which currently does not support [Fragments](https://reactjs.org/docs/fragments.html#short-syntax) (they do in Preact X). Because of this, a dummy container in markup needs to be created so that the Preact component `<TagsFollowed />` can be mounted.